Mono Browser wasm Release failures on runtime rolling build

### Description

Last rolling build of runtime fails with following exceptions 

Unhandled Exception:
StackOverflowException
[ERROR] FATAL UNHANDLED EXCEPTION: System.StackOverflowException: The requested operation caused a stack overflow.
at ExplicitTailCallNoSO.CalculateSumWithTailCall(Int32 n, Int32 sum

or

Unhandled Exception:
StackOverflowException
[ERROR] FATAL UNHANDLED EXCEPTION: System.StackOverflowException: The requested operation caused a stack overflow.
at Class1.Recurse4(Int32 depth, Object o1, Object o2)
for more info please look  https://dev.azure.com/dnceng/public/_build/results?buildId=1790753&view=ms.vss-test-web.build-test-results-tab

### Reproduction Steps

Check last rolling build of runtime https://runfo.azurewebsites.net/view/build/?number=1790753

### Expected behavior

Tests should pass

### Actual behavior

[ERROR] FATAL UNHANDLED EXCEPTION: System.StackOverflowException

### Regression?

_No response_

### Known Workarounds

_No response_

### Configuration

_No response_

### Other information

_No response_

--- a/src/tests/issues.targets
+++ b/src/tests/issues.targets
@@ -3285,6 +3285,10 @@
 
 
     <ItemGroup Condition=" '$(TargetArchitecture)' == 'wasm' " >
+        <ExcludeList Include="$(XunitTestBinBase)/JIT/Directed/tailcall/**">
+            <Issue>https://github.com/dotnet/runtime/issues/69517</Issue>
+        </ExcludeList>
+
         <ExcludeList Include="$(XunitTestBinBase)/JIT/Intrinsics/TypeIntrinsics_r/**">
             <Issue>https://github.com/dotnet/runtime/issues/54867</Issue>
         </ExcludeList>

--- a/src/tests/issues.targets
+++ b/src/tests/issues.targets
@@ -3288,6 +3288,9 @@
         <ExcludeList Include="$(XunitTestBinBase)/JIT/Directed/tailcall/**">
             <Issue>https://github.com/dotnet/runtime/issues/69517</Issue>
         </ExcludeList>
+        <ExcludeList Include="$(XunitTestBinBase)/JIT/Directed/IL/Tailcall/ExplicitTailCallNoSO.ilproj">
+            <Issue>https://github.com/dotnet/runtime/issues/69517</Issue>
+        </ExcludeList>
 
         <ExcludeList Include="$(XunitTestBinBase)/JIT/Intrinsics/TypeIntrinsics_r/**">
             <Issue>https://github.com/dotnet/runtime/issues/54867</Issue>

--- a/src/tests/issues.targets
+++ b/src/tests/issues.targets
@@ -3288,7 +3288,7 @@
         <ExcludeList Include="$(XunitTestBinBase)/JIT/Directed/tailcall/**">
             <Issue>https://github.com/dotnet/runtime/issues/69517</Issue>
         </ExcludeList>
-        <ExcludeList Include="$(XunitTestBinBase)/JIT/Directed/IL/Tailcall/ExplicitTailCallNoSO*">
+        <ExcludeList Include="$(XunitTestBinBase)/JIT/Directed/IL/Tailcall/ExplicitTailCallNoSO/*">
             <Issue>https://github.com/dotnet/runtime/issues/69517</Issue>
         </ExcludeList>
 

--- a/src/tests/issues.targets
+++ b/src/tests/issues.targets
@@ -3288,7 +3288,7 @@
         <ExcludeList Include="$(XunitTestBinBase)/JIT/Directed/tailcall/**">
             <Issue>https://github.com/dotnet/runtime/issues/69517</Issue>
         </ExcludeList>
-        <ExcludeList Include="$(XunitTestBinBase)/JIT/Directed/IL/Tailcall/ExplicitTailCallNoSO.ilproj">
+        <ExcludeList Include="$(XunitTestBinBase)/JIT/Directed/IL/Tailcall/ExplicitTailCallNoSO*">
             <Issue>https://github.com/dotnet/runtime/issues/69517</Issue>
         </ExcludeList>
 

